src/components/Farm/FarmSummary.tsx

```diff
@@ -61,86 +15,12 @@ const FarmSummary = ({ poolsInfo, tokenPrice }: FarmSummaryProps) => {
   const { t } = useTranslation()
 
   return (
-    <DashboardRow>
-      <InfoRow>
-        <StakeCard>
-          <InfoTitleRow>
-            <TYPE.subHeader
-              style={{
-                letterSpacing: '0.1em',
-                fontWeight: 800,
-                textTransform: 'uppercase',
-                fontSize: '0.75rem',
-                justifySelf: 'flex-start'
-              }}
-            >
-              {t('poolSummaryStakeable')}
-            </TYPE.subHeader>
-          </InfoTitleRow>
-          <Row>
-            <TYPE.body
-              style={{ marginTop: '0.1rem', fontWeight: 600, fontSize: '1.5rem', justifySelf: 'flex-start' }}
-              id="text-stakeable-value"
-            >
-              {summary.stakeableValue}
-            </TYPE.body>
-          </Row>
-        </StakeCard>
-        <StakeCard>
-          <InfoTitleRow>
-            <TYPE.subHeader
-              style={{
-                letterSpacing: '0.1em',
-                fontWeight: 800,
-                textTransform: 'uppercase',
-                fontSize: '0.75rem',
-                justifySelf: 'flex-start'
-              }}
-            >
-              {t('poolSummaryStaked')}
-            </TYPE.subHeader>
-          </InfoTitleRow>
-          <Row>
-            <TYPE.body
-              style={{ marginTop: '0.1rem', fontWeight: 600, fontSize: '1.5rem', justifySelf: 'flex-start' }}
-              id="text-staked-value"
-            >
-              {summary.stakedValue}
-            </TYPE.body>
-          </Row>
-        </StakeCard>
-      </InfoRow>
-      <Row>
-        <EarnCard>
-          <HideSmall>
-            <Row>
-              <img style={{ marginBottom: '0.5rem' }} width={'85px'} src={HaloHalo} alt="xRNBW" />
-            </Row>
-          </HideSmall>
-          <EarnedTitleRow>
-            <TYPE.subHeader
-              style={{
-                letterSpacing: '0.1em',
-                fontWeight: 800,
-                textTransform: 'uppercase',
-                fontSize: '0.75rem',
-                justifySelf: 'flex-start'
-              }}
-            >
-              {t('poolSummaryHaloEarned')}
-            </TYPE.subHeader>
-          </EarnedTitleRow>
-          <Row>
-            <TYPE.body
-              style={{ marginTop: '0.1rem', fontWeight: 600, fontSize: '1.5rem', justifySelf: 'flex-start' }}
-              id="text-halo-earned"
-            >
-              {summary.haloEarned}
-            </TYPE.body>
-          </Row>
-        </EarnCard>
-      </Row>
-    </DashboardRow>
+    <div className="w-full flex flex-col md:flex-row space-y-2 md:space-y-0 md:space-x-2 pb-2">
+      <StakeCard title={t('totalPoolValue')} value="5000.03" />
```

Comment:
@geonellov i see this is hardcoded. Looking at figma https://www.figma.com/file/lqpLD77mM2olJNrFzuD4Ul?node-id=7268:3943#148922421 it should be "Total Value Locked" which is the sum of all the USD values of `Total Pool Value` column

Or have we done this in another PR?

Reply:
yep, it's done in another PR @ctverceles

Reply:
Oh yeah I was trying to find this too, because we are missing the $ on the UI